README.md

```diff
@@ -126,8 +126,13 @@ These composites are also the motivation for the short aliases of payloads, as t
 For more examples, please see the [sample command](https://github.com/EluciusFTW/SpectreCoff/blob/main/src/spectrecoff-cli/commands/Output.fs).
 
 ### Modules
+This Bar Chart (created using SpectreCoff) breaks down our progress taking over the Spectre.Console modules.
+Much is already usable, but there is still a way to go. 
+
+![](assets/progress.png)
```

Comment:
Also, the screenshot 'hangs in the air' - why not create a new command, `dotnet run progress` that has this as output and leave the old example as is. Then in the screenshot, include the calling line/prompt where you trigger the command.

Reply:
And in the progress, since the title sais they are SpectreCoff modules, I'd call them by correct name (no blanks). Also `BarChart + Breakdown Chart -> Chart`, right?

Reply:
Did what you asked, it's a good idea :)